cmd/controller/main.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"knative.dev/pkg/configmap"
+	"knative.dev/pkg/controller"
+	"knative.dev/pkg/injection/sharedmain"
+	"knative.dev/pkg/signals"
+
+	"knative.dev/net-http01/pkg/challenger"
+	"knative.dev/net-http01/pkg/reconciler/certificate"
+)
+
+func main() {
+	// Uncomment this to use the Let's Encrypt Staging environment.
```

Comment:
Turn this into flag?

Reply:
I think I'd rather move it into a configmap in follow-up?

Reply:
sgtm

Reply:
Opened: https://github.com/knative/net-http01/issues/10